mousestyles/data/__init__.py

```
@@ -181,3 +184,103 @@ def load_movement(strain, mouse, day):
+    """
+    Return a pandas.DataFrame object of project movement and interval
+    data for the specified combination of strain, mouse and day.
+
+    There are 4 + len(features) columns in the dataframe: t: Time
+    coordinates (in seconds) x: X coordinates indicating the
+    left-right position of the cage y: Y coordinates indicating the
+    front-back position of the cage isHB: Boolean indicating whether
+    the point is in the home base or not Additonal columns taking
+    their names from features: Boolean indicating whether the time
+    point is in an interval of behavior of the given feature.
+
+    Parameters
+    ----------
+    strain: int
+        nonnegative integer indicating the strain number
+    mouse: int
+        nonnegative integer indicating the mouse number
+    day: int
+        nonnegative integer indicating the day number
+    features: list (or other iterable) of strings
+        list of features from {"AS", "F", "IS", "M_AS", "M_IS", "W"}
+
+    Returns
+    -------
+    movement : pandas.DataFrame CT, CX, CY
+        coordinates, home base status, and feature interval information
+        for a given srain, mouse and day
+
+    Examples
+    --------
+    >>> m1 = load_movement(1, 1, 1)
+    >>> m2 = load_movement_and_intervals(1, 1, 1, []) # don't add any features
+    >>> np.all(m1 == m2)
+    True
+    >>> m3 = load_movement_and_intervals(1, 1, 1, ['AS'])
+    >>> m3.shape[1] == m1.shape[1] + 1 # adds one column
+    True
+    >>> m3.shape[0] == m1.shape[0] # same number of rows
+    True
+    >>> m3[29:32]
+                t      x       y   isHB     AS
+    29  56448.333 -6.289  34.902  False  False
+    30  56448.653 -5.509  34.173   True   True
+    31  56449.273 -5.048  33.284   True   True
+    """
+    movements = load_movement(strain, mouse, day)
+    for f in features:
+        intervals = load_intervals(feature=f)
+        mouse_intervals = intervals[(intervals['strain'] == strain) &
```

What if they specify a mouse/day/strain that does not exist in this dataframe? You should throw an exception here I think...